client/main.lua

```diff
@@ -98,330 +84,357 @@ function AlertPolice()
     Citizen.Wait(500)
 end
 
-RegisterNetEvent('AttackTransport:InfoForLspd', function(x, y, z)
-    if PlayerJob ~= nil and PlayerJob.name == 'police' then
-
-        if PoliceBlip == 0 then
-            PoliceBlip = 1
-            local blip = AddBlipForCoord(x, y, z)
-            SetBlipSprite(blip, 67)
-            SetBlipScale(blip, 1.0)
-            SetBlipColour(blip, 2)
-            BeginTextCommandSetBlipName("STRING")
-            AddTextComponentString('Assault on the transport of cash')
-            EndTextCommandSetBlipName(blip)
-            SetNewWaypoint(x, y)
-            Citizen.Wait(10000)
-            RemoveBlip(blip)
-            PoliceBlip = 0
-        end
-
-        local PoliceCoords = GetEntityCoords(PlayerPedId(), false)
-        local PoliceDist = #(PoliceCoords - vector3(x, y, z))
-        if PoliceDist <= 4.5 then
-            local dict = "anim@mp_player_intmenu@key_fob@"
-
-            RequestAnimDict(dict)
-            while not HasAnimDictLoaded(dict) do
-                Citizen.Wait(100)
-            end
-            if SilenceAlarm == 0 then
-                hintToDisplay('Press ~INPUT_DETONATE~ to silence the alarm')
-                SilenceAlarm = 1
-            end
-            if IsControlPressed(0, 47) and GuardsDead == 1 then
-
-                TaskPlayAnim(PlayerPedId(), dict, "fob_click_fp", 8.0, 8.0, -1, 48, 1, false, false, false)
-                TriggerEvent('AttackTransport:CleanUp')
-                RemoveBlip(TruckBlip)
-                Citizen.Wait(500)
-            end
-        end
-
-    end
+RegisterNetEvent('AttackTransport:InfoForLspd')
+AddEventHandler('AttackTransport:InfoForLspd', function(x, y, z)
```

Comment:
You can remove AddEventHandler and pass the event handler directly to RegisterNetEvent